src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/index.tsx

```diff
@@ -77,6 +78,16 @@ function _defaultComponents() {
     CodeTabs,
     pre,
     a: DocsAnchor,
+    /**
+     * In /docs, we want to hide image borders by default for now,
```

Comment:
Added this comment for clarity on border behaviour in docs. This thought also prompted me to make [[discuss] reconcile MDX components for /docs and /tutorials](https://app.asana.com/0/1201987349274776/1202052090505082/f), which I think is something that might be more of a "when" discussion than "if".